sdv/metadata/table.py

```diff
@@ -157,10 +157,33 @@ class Table:
         ('id', 'string'): 'str'
     }
 
-    def _get_faker(self, category):
-        """Return the faker object to anonymize data.
+    def _get_fake_values(self, field_metadata, num_values):
+        faker = self._get_faker(field_metadata)
+        return (
+            self._get_faker_method(faker, field_metadata['pii_category'])()
+            for _ in range(num_values)
+        )
+
+    def _get_faker(self, field_metadata):
+        """Return the faker object with localisaton set if specified in field_metadata.
 
         Args:
+            field_metadata (dict):
+                Metadata for field to read localisation from if set in `pii_locales`.
+
+        Returns:
+            Faker object:
+                The Faker object to anonymize the data in the field using its functions.
+        """
+        pii_locales = field_metadata.get('pii_locales', None)
+        return Faker(locale=pii_locales) if pii_locales is not None else Faker()
```

Comment:
This is removed, but i kept the local `pii_locales` varialbe for clarity.
If wanted, this could also be moved inside the call:
```
Faker(locale=field_metadata.get('pii_locales', None))
```